### PR TITLE
[VL] Fix native cmake include path

### PR DESCRIPTION
**Pull request by Yohahaha**

### CODE REVIEW

**Comment by Yohahaha** [comment]
@rui-mo @PHILO-HE @marin-ma could someone help review this patch? it helps build index for my clion.

**Review by PHILO-HE — APPROVED** [review]
LGTM! Thanks!

File: cpp/velox/CMakeLists.txt [hunk 1]
```diff
@@ -275,12 +275,9 @@ target_include_directories(velox PUBLIC
     ${JNI_INCLUDE_DIRS}
     ${CMAKE_CURRENT_SOURCE_DIR}
     ${root_directory}/src
```
Comment (marin-ma) [inline cpp/velox/CMakeLists.txt:277]: Looks like this include path has been deprecated for a long time. Could you help to remove it as well?

Comment (Yohahaha) [reply cpp/velox/CMakeLists.txt:277]: ok, I wil refine it.

**Comment by marin-ma** [comment]
Thanks!

**Review by PHILO-HE — APPROVED** [review]
Thanks!

**Pull request merged**